.editorconfig

```diff
@@ -53,9 +53,6 @@ indent_size = 2
 indent_size = 4
 tab_width = 4
 
-# New line preferences
```

Comment:
Is this a change from default? Please annotate it rather than deleting.

Reply:
Looks like my comment was removed when I pushed! I was saying that for some reason this was duplicate with a different value for C#. The entry exists and is set to true for all files at the beginning of the file.